test/unit/specs/components/scroll-button.spec.js

```diff
@@ -42,7 +41,6 @@ describe('Scroll button', () => {
     const button = wrapper.find('button');
     expect(button).toBeDefined();
     button.trigger('click');
-    expect(mockMethods.scrollToTop).toHaveBeenCalled();
+    expect(mockMethods).toHaveBeenCalled();
```

Comment:
This test will not work since the name of the function being called in the component is "scrollToTop" instead of "scrollTo".

Reply:
While doing `npm run test`, the component ScrollButton.vue shows line number 36 as uncovered which is calling the `window.scrollTo` function. I was looking at some answers on how to access it and found `global` but that seems to not work. 
Could you recommend some way to implement this?